neo/rawio/neuralynxrawio/neuralynxrawio.py

```diff
@@ -183,33 +213,24 @@ def _parse_header(self):
         event_annotations = []
 
         if self.rawmode == "one-dir":
-            filenames = sorted(os.listdir(self.dirname))
-            dirname = self.dirname
+            filenames = os.listdir(self.dirname)
```

Comment:
I am realising only now that this little has an bi big impact because the file are unsorted and so channel are not ordered

Reply:
See this https://github.com/SpikeInterface/spikeinterface/issues/3248

Reply:
You think this is Anthony's issue? This was just merged yesterday, so I guess we could confirm that the issue showed up, but makes sense that could be the problem. You plan to just open the PR?

Reply:
Oh, sorry for missing this issue. Do we also want to sort `include_filenames`? One edge case would be the user using `os.listdir` to collect the files and pass it to `include_filenames`.

Reply:
No wory. @Antho2422 did a quick fix.
For include_filenames, I think we do not need the sorter() because it makes that the user can providde a list in the desired order. no ?